### ISSUE TITLE
Issue with Plugin 153214, possibly because of the extra brackets in the name?

### ISSUE THREAD

**Issue by dil-ddsouza** [tenable/integration-jira-cloud#144]
Hi Steve,

I noticed an issue that the script fails when it comes to the plugin '153214', specifically transitioning the issues in Jira from To-Do to Done.

Below is the error I see:
```
2021-09-28 04:37:46,836 tenable_jira.jira.Jira INFO UPDATED VULN-148164 [153214] Security Updates for Microsoft Internet Explorer OOB (Sept 2021)
2021-09-28 04:37:46,836 tenable_jira.jira.Jira DEBUG Request:{"method": "PUT", "url": "https://jira.atlassian.net/rest/api/3/issue/620669", "params": {"notifyUsers": "true", "overrideScreenSecurity": "false", "overrideEditableFlag": "false"}, "body": {"fields": {"project": {"key": "VULN"}, "issuetype": {"id": 18}, "customfield_16098": ["CVE-2021-40444"], "customfield_16099": "8.3", "customfield_16100": "2021-09-07T18:00:00.000+0000", "customfield_16101": "7.2", "customfield_16102": "8.8", "customfield_16103": "8.4", "customfield_16104": "153214", "customfield_16105": "Windows", "customfield_16106": "Security Updates for Microsoft Internet Explorer OOB (Sept 2021) (deprecated)", "customfield_16107": "High", "customfield_16124": "9.8", "summary": "[153214] Security Updates for Microsoft Internet Explorer OOB (Sept 2021) (deprecated)", "description": {"version": 1, "type": "doc", "content": [{"type": "heading", "attrs": {"level": 1}, "content": [{"type": "text", "text": "Description"}]}, {"type": "paragraph", "content": [{"type": "text", "text": "This plugin is a work-around and is being deprecated due other superceded Microsoft Security patches.  See Nessus Plugin IDs: 153374, 153214, 153372, 153373, 153375, 153377, 153381, 153383"}]}, {"type": "heading", "attrs": {"level": 1}, "content": [{"type": "text", "text": "Solution"}]}, {"type": "paragraph", "content": [{"type": "text", "text": ""}]}]}}}}
2021-09-28 04:37:47,260 urllib3.connectionpool DEBUG https://jira.atlassian.net:443 "PUT /rest/api/3/issue/620669?notifyUsers=true&overrideScreenSecurity=false&overrideEditableFlag=false HTTP/1.1" 400 None
2021-09-28 04:37:47,264 restfly.errors.BadRequestError ERROR [400: PUT] https://jira.atlassian.net/rest/api/3/issue/620669?notifyUsers=true&overrideScreenSecurity=false&overrideEditableFlag=false body=b'{"errorMessages":["INVALID_INPUT"],"errors":{}}'
```

Could it be because of the plugin name? As the error message is suggesting an error in the Input. So my guess is the plugin name, as I think the script expects only on pair of '( )', however this plugin has 3 pairs of '( )'? 

OR 

it is to do with the change of task name, because the original task created had the plugin name as: 
[153214] Security Updates for Microsoft Internet Explorer OOB (Sept 2021)

So now due to the change of plugin name, it's struggling to transition the issues maybe? 

Plugin Name: Security Updates for Microsoft Internet Explorer OOB (Sept 2021) (deprecated) (153214)

**Comment by SteveMcGrath** [comment]
It shouldn't matter TBH.  The plugin is a deprecated one however.  Can you check to see if you have any of them in the Jira project?

Whats unfortunate is that the error message gives no reasoning as to why it has failed.  I'll look into this, however the quick workaround would be to simply close any instance of plugin 153214 manually.

**Comment by SteveMcGrath** [comment]
so I can confirm that modifying the summary field doesn't seem to create that issue.

**Comment by SteveMcGrath** [comment]
Found the issue.  it appears that Jira barfs when a content paragraph has an empty string.

**Comment by SteveMcGrath** [comment]
should be resolved in 1.2.1

**Comment by dil-ddsouza** [comment]
Thanks Steve, I'll try it out in 1.2.1
